### PR TITLE
[Enhancement][RHEL/7] Add RHEL/7 STIG Reference Identifiers

### PR DESCRIPTION
**Pull request by redhatrises**
- Add RHEL/7 STIG identifier
- Add RHEL/7 OS URI Link

### CODE REVIEW

**Comment by redhatrises** [comment]
Anyone know why we decided to make the `RHEL/7` STIG ID as a `reference` element rather than an `ident` element like it is in `RHEL/6`?

File: RHEL/7/transforms/shorthand2xccdf.xslt [hunk 1]
```diff
@@ -188,8 +188,18 @@
...
+          </xsl:if>
         </xsl:attribute>
-        <xsl:value-of select="normalize-space($refitem)" />
+        <xsl:choose>
+          <xsl:when test="name() = 'stigid'">
+            <xsl:value-of select="normalize-space(concat('RHEL-07-', .))" />
+          </xsl:when>
+        <xsl:otherwise>
+          <xsl:value-of select="normalize-space($refitem)" />
+        </xsl:otherwise>
```
Comment (mpreisler) [inline RHEL/7/transforms/shorthand2xccdf.xslt]: It's a minor nitpick but please fix the indent, it was hard for me to figure out the logic.

**Comment by redhatrises** [comment]
Sorry about that @mpreisler. Indentation has been fixed.

**Comment by mpreisler** [comment]
> Anyone know why we decided to make the RHEL/7 STIG ID as a reference element rather than an ident element like it is in RHEL/6?

No idea. Do you want to get that resolved in this PR or should I merge it?

**Comment by redhatrises** [comment]
>> Anyone know why we decided to make the RHEL/7 STIG ID as a reference element rather than an ident element like it is in RHEL/6?

>No idea. Do you want to get that resolved in this PR or should I merge it?

@mpreisler go ahead and merge. I guess this is more of a question for @shawndwells when he is back.

**Pull request merged**